tests/vxingest/netcdf_to_cb/test_unit_metar_obs_netcdf.py

```diff
@@ -153,18 +155,21 @@ def test_vxingest_get_file_list(tmp_path):
             "projection": "lambert_conformal_conic",
             "subset": "metar",
             "type": "DF",
-            "url": str(tmp_path / "f_fred_01"),
+            "url": str(tmp_path / "1820013010000"),
         }
         vx_ingest.collection.upsert("DF:metar:grib2:HRRR_OPS:f_fred_01", df_record)
+        # make a ".prev" and a ".tmp" directory to see if they get filtered out
+        Path(tmp_path / ".prev").mkdir()
+        Path(tmp_path / ".tmp").mkdir()
         # order is important to see if the files are getting returned sorted by mtime
-        Path(tmp_path / "f_fred_01").touch()
-        Path(tmp_path / "f_fred_02").touch()
-        Path(tmp_path / "f_fred_04").touch()
-        Path(tmp_path / "f_fred_05").touch()
-        Path(tmp_path / "f_fred_03").touch()
-        Path(tmp_path / "f_1_fred_01").touch()
-        Path(tmp_path / "f_2_fred_01").touch()
-        Path(tmp_path / "f_3_fred_01").touch()
+        Path(tmp_path / "1820013010000").touch()
+        Path(tmp_path / "1820013020000").touch()
+        Path(tmp_path / "1820013040000").touch()
+        Path(tmp_path / "1820013050000").touch()
+        Path(tmp_path / "1820013030000").touch()
+        Path(tmp_path / "1820014010000").touch()
+        Path(tmp_path / "1820015010000").touch()
+        Path(tmp_path / "1820016010000").touch()
```

Comment:
It'd be good to add the following file as a test case to make sure we're doing the right thing when there's a valid filename in a hidden directory.

```python
Path(tmp_path / "1820013010000").touch()
Path(tmp_path / ".prev" / "1820013010000").touch()
Path(tmp_path / ".tmp" / "1820013010000").touch()
```